views/partials/form.handlebars

```diff
@@ -1 +1,42 @@
-Edit topic form
\ No newline at end of file
+Edit topic form
+
+<div class="row">
+    <p>Create a new Blog post</p>
```

Comment:
This seems to have been copied over from the blog post example. Could you please look over it and make sure there are no references to "blog" or "post"?